### PR TITLE
made a comment for demonstration purposes

### PR DESCRIPTION
**Pull request by wescottj98**

### CODE REVIEW

File: mission/config/subconfigs/vehicle_respawn_info.hpp [hunk 1]
```diff
@@ -38,8 +38,12 @@ class wreck_long
 {
 	respawnType = "WRECK";
 	time = 60;
+
```
Comment (dijksterhuis) [inline mission/config/subconfigs/vehicle_respawn_info.hpp:41]: Please could you remove this extra newline for me?

File: mission/config/subconfigs/vehicle_respawn_info.hpp [hunk 2]
```diff
@@ -38,8 +38,12 @@ class wreck_long
...
 };
 
+
+// comment from dev 101 meeting 3/16/2024
```
Comment (dijksterhuis) [inline mission/config/subconfigs/vehicle_respawn_info.hpp:45]: is this US date format?

**Review by dijksterhuis — CHANGES_REQUESTED** [review]
remove the new line and just confirm the date format for me please?

File: mission/config/subconfigs/vehicle_respawn_info.hpp (hunk 2 above)
Comment (wescottj98) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:45]: Yes

File: mission/config/subconfigs/vehicle_respawn_info.hpp (hunk 1 above)
Comment (wescottj98) [reply mission/config/subconfigs/vehicle_respawn_info.hpp:41]: No

**Pull request closed without merging**